### ISSUE TITLE
[Bug]: Garry's mod Egg

### ISSUE THREAD

**Issue by dylanc24** [parkervcp/eggs#1740]
### Panel Version

1.9.2

### Wings Version

1.6.4

### Service

garry's mod

### Modified

No, I did not modify the egg

### Expected Behavior

I thought I would find these two options to install the 64-bit version of the garry'smod server and validate the files. Options which are in fact available on the version of the egg present on github but not on the panel version that is created after installation.
![screenshoot](https://user-images.githubusercontent.com/18116295/177401006-9b9bb02e-1d78-472f-8527-d6aee8c078d6.png)
g

Moreover with the latest egg version from the github, when you activate the 64 bits beta release it didn't automaticaly switch the starting command from "./srcds_run" to "./srcds_run_x64"

### Actual Behavior

So the actual version of the egg in the panel is without the 2 options and if you install the latest version and install a new serv with the 64 bits options enabled it didn't automaticaly switch the startup command. 

### Steps To Reproduce

Install a new Garry's Mod server via the base garry's mod egg created after the installation. 

### Install logs

https://pteropaste.com/fe591h

**Comment by Software-Noob** [comment]
I don't see anything here that's considered a bug, but your issue is hard to follow as you've typed out so many different things, while a bug report should be about a single issue. 

This egg repository and default eggs shipped with the panel are two seperate things. This repository is a collection of eggs for Pterodactyl. Some of these changes are pushed upstream to the default eggs when deemed fit and tested. 

Eggs themselves can't modify startup parameters and renaming the binary would be misleading. I don't see a way to do use the right binary for the user without multiple checks.

**Comment by lilkingjr1** [comment]
Closing as stale. Please comment to reopen if you wish.
